packages/ipa/env/run.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 
-PWD=`pwd`/`dirname ${BASH_SOURCE}`
+CWD=`pwd`/`dirname ${BASH_SOURCE}`
 
-npx coffee $PWD/ipa/start.coffee
-lxc exec freeipa bash <<EOF
-cd /nikita/packages/ipa
-npm test
-EOF
+npx coffee $CWD/ipa/start.coffee
+lxc exec freeipa --cwd /nikita/packages/ipa npx mocha 'test/**/*.coffee'
```

Comment:
I prefer how it was before, I found it easier to read and more flexible and generic, for example the same pattern can be used when more commands are required, we can easily write additional commands for debugging purposes...

Reply:
Done. Changed everywhere